lib/ProductOpener/Display.pm

```diff
@@ -1560,7 +1565,7 @@ oTable = \$('#tagstable').DataTable({
 	order: [[ 1, "desc" ]],
 	columns: [
 		null,
-		{ "searchable": false }
+		{ "searchable": false } $extra_column_searchable
```

Comment:
You might have forgotten to remove the content now inside the variable.

Reply:
looks good to me

Reply:
Thanks!

@nicolasleger The idea was that `$extra_column_searchable` is either empty or contains the entry for the third column. Not the cleanest way to implement this - I'll admit 😄 - but it should work. 😉

Reply:
Ok no problem so ! I was in doubt, sorry. Good 👍🙂

Reply:
No problem. Thanks for checking! :)